daemon/algod/server.go

```diff
@@ -229,6 +229,9 @@ func (s *Server) Initialize(cfg config.Local, phonebookAddresses []string, genes
 			NodeExporterPath:          cfg.NodeExporterPath,
 		})
 
+	var algodEnvironmentAlgodVersionCounter = metrics.MakeCounter(metrics.MetricName{Name: "algod_environment_algod_version", Description: "Version of the Algod binary"})
```

Comment:
```suggestion
	var algodEnvironmentAlgodVersionCounter = metrics.MakeCounter(metrics.MetricName{Name: "algod_version", Description: "Version of the Algod binary"})
```

Reply:
@algorandskiy Happy to change, but reasoning for "environment" namespace is to add information about the runtime environment, this could include things such as memory pressure etc.

There's many ways to skin a cat, however having it here provides an easy way to pull data into existing infrastructure if you already pull the metrics, and it's in line with what node_exporter would export for any standard setup. If added to the status endpoint anyone would have to augment metrics gathering to include hitting this endpoint for environment related information.

In my opinion the /status endpoint would be better to use for reporting about the healthy status of the service (frequently they are referred to as health checks), and not the environment of the service. Generally speaking I would argue that status/health check endpoints are better suited for reporting on things such healthy connection to data stores, if healthy state is present, etc. all to determine if you want to have a node be part of any clusters.   Whereas application layer information about runtime environment/data fed into business KPI's are would be better suited to go into your observability platform as you there may want to index/log/query/graph/search the information.